frontend/components/footer.js

```diff
@@ -47,13 +39,19 @@ const Footer = ({ baseUrl }) => (
         alt="chat on Discord"
       />
     </p>
+
     <p>
-      <a href="https://github.com/h5bp/lazyweb-requests/issues/150">This</a> is
-      where the current server got started.
+      What is your favorite badge service to use?
+      <br />
+      <a href="https://github.com/badges/shields/blob/master/CONTRIBUTING.md">
+        Tell us
+      </a>{' '}
+      and we might bring it to you!
     </p>
 
-    <p>
-      <small>:wq</small>
+    <p className="spaced-row">
+      <a href="https://status.shields.io/">Status</a>
```

Comment:
new toys! :gift: 
Is https://status.shields.io/ supposed to have any content yet? I just get a loading animation forever

Reply:
Ah, gosh I didn't know you hadn't seen this! I thought this probably need more publicity. 😝 

This is what I'm seeing, though it's being a bit slow today.

<img width="940" alt="screen shot 2018-11-09 at 1 57 04 pm" src="https://user-images.githubusercontent.com/1487036/48282318-5e321c80-e427-11e8-88d8-502e9cf102af.png">

This had been hosted at status.shields-server.com but I switched it over a couple months back when I got access to the Shields Cloudflare account.